include/TPP/Dialect/Xsmm/XsmmOps.td

```diff
@@ -9,8 +9,8 @@
 #ifndef TPP_XSMM_OPS
 #define TPP_XSMM_OPS
 
-include "XsmmDialect.td"
-include "XsmmAttr.td"
+include "TPP/Dialect/Xsmm/XsmmDialect.td"
+include "TPP/Dialect/Xsmm/XsmmAttr.td"
```

Comment:
Why is this necessary?

Reply:
Keep things consistent with MLIR and how we do in other files.